### PR TITLE
feat: added interactive examples

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

**Comment by martijnrusschen** [comment]
I noticed `yarn start` is giving an error. I'm not sure if this is the same in master, but wanted to point this out:
```
➜  react-datepicker git:(pr/1864) yarn run start
yarn run v1.15.2
$ cross-env MODULES=false node server.js
module.js:550
    throw err;
    ^

Error: Cannot find module '/Users/martijn/Development/react-datepicker/server.js'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

File: package.json [hunk 1]
```diff
@@ -105,14 +104,16 @@
...
   },
   "scripts": {
-    "eslint": "eslint --ext .js,.jsx src test docs-site/src",
+    "eslint": "eslint --ext .js,.jsx src test",
     "flow": "flow",
     "precommit": "lint-staged",
-    "sass-lint": "sass-lint --config .sass-lint.yml 'src/stylesheets/*.scss, docs-site/src/*.scss' -i 'docs-site/src/higlight.scss, docs-site/src/reset.scss'",
+    "sass-lint": "sass-lint --config .sass-lint.yml 'src/stylesheets/*.scss'",
     "lint": "run-p eslint flow sass-lint",
     "start": "cross-env MODULES=false node server.js",
```
Comment (martijnrusschen) [inline package.json]: This command doesn't work anymore.

Comment (mxschmitt) [reply package.json]: for me it works. What does not work exactly?

Comment (martijnrusschen) [reply package.json]: See error I posted earlier. I guess we're trying to boot server.js which is removed in this PR

Comment (mxschmitt) [reply package.json]: This error I've fixed here: https://github.com/Hacker0x01/react-datepicker/pull/1864/commits/46cbed3a23bdac21d5d2add55d475ccb3c530253

**Comment by martijnrusschen** [comment]
Working well now. It took me a bit before I noticed that I could edit the code examples. Should we make it look more like a real editor?

**Comment by mxschmitt** [comment]
> Working well now. It took me a bit before I noticed that I could edit the code examples. Should we make it look more like a real editor?

Maybe add some kind of edit icon in the top right corner? https://fontawesome.com/icons/edit?style=regular

Or write a sentence in the top about it.

**Comment by martijnrusschen** [comment]
Yep, I think and icon would do the trick indeed.

**Comment by mxschmitt** [comment]
Maybe something for a follow up PR I think. Also have to see if everything works with deployment when we merge to master.

**Comment by mxschmitt** [comment]
Deployment of course is broken 😄 Have you set the gh-pages branch in the Repository GitHub Pages settings? (not the docs-site sub directory)

**Comment by martijnrusschen** [comment]
I think it's the CNAME file that was removed. Adding it back now to see if that fixes the issue.

**Comment by mxschmitt** [comment]
Good catch!

**Comment by mxschmitt** [comment]
Works when you bypass the cache: https://reactdatepicker.com/?1 Thanks!

**Comment by martijnrusschen** [comment]
Yep, works again.

**Pull request merged**